.github/workflows/release-notes.yaml

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright (C) 2023 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: release-notes
+
+on:
+  workflow_dispatch:
+    inputs:
+      milestone:
+        description: 'Milestone to generate release notes from'
+        required: true
```

Comment:
maybe need to add placeholder (example in the description) with the format of the expected value

is it: 1.0.0, v1.0.0 or milestone number like 18 for 1.1.0 https://github.com/containers/podman-desktop/milestone/18